core/src/main/java/nl/nn/adapterframework/http/cxf/SOAPProviderBase.java

```diff
@@ -130,7 +130,7 @@ public SOAPMessage invoke(SOAPMessage request) {
 						sessionKey.setValue("attachment" + i);
 						attachment.addSubElement(sessionKey);
 						Message rawContent = new Message(attachmentPart.getRawContent());
-						rawContent.closeOnCloseOf(pipelineSession);
+						//rawContent.closeOnCloseOf(pipelineSession);
```

Comment:
I suggest to replace these lines with: 
`Message rawContent = new Message(attachmentPart.getRawContentBytes());`
The attachments are not really coming from the HttpRequest.inputstream anymore, because they can be accessed in any order.